### PR TITLE
keyboardNav: Generate commentsLinkNumbers dynamically

### PR DESCRIPTION
**Pull request by larsjohnsen**
- So that `Event.preventDefault` won't be invoked when `commentsLinkNumbers` is disabled
- Provide alternative mode to invert `commentsLinkToggleExpando` action (toggle expando vs open in new tab)

Fixes https://www.reddit.com/r/RESissues/comments/518czp/bug_commentslinknumbers_interferes_with_single/

### CODE REVIEW

**Comment by andytuba** [comment]
I wonder how many people have changed the default keybindings for link#.. Probably too few to worry about.

**Comment by larsjohnsen** [comment]
@andytuba The links keybindings has been `noconfig` since v4.6.0, so probably very few.

File: lib/modules/keyboardNav.js [hunk 1]
```diff
@@ -802,9 +683,32 @@ const drawHelp = _.once(() => {
...
+	const keyArrays = [];
+
+	function addArray(key, index) {
+		keyArrays.push({
+			type: 'keycode',
+			value: [key, false, false, false], // number
+			callback() { commentLink(index); },
+		}, {
+			type: 'keycode',
+			value: [key, true, false, false], // shift-number
```
Comment (erikdesjardins) [inline lib/modules/keyboardNav.js]: This is alt; the third bool is for shift

Comment (larsjohnsen) [inline lib/modules/keyboardNav.js]: Yeah; forgot to update that comment. Will update.

Comment (erikdesjardins) [inline lib/modules/keyboardNav.js]: The option desc needs an update too I believe

**Pull request merged**